src/cascadia/TerminalControl/SearchBoxControl.xaml

```diff
@@ -0,0 +1,187 @@
+                    
+                    <SolidColorBrush x:Key="ToggleButtonBackgroundPointerOver" Color="#DADADA"/>
+                    <SolidColorBrush x:Key="ToggleButtonForegroundPointerOver" Color="#000000"/>
+                    <SolidColorBrush x:Key="ToggleButtonBorderBrushPointerOver" Color="Transparent"/>
+
+                    <SolidColorBrush x:Key="ToggleButtonBackgroundPressed" Color="#B8B8B8"/>
+                    <SolidColorBrush x:Key="ToggleButtonForegroundPressed" Color="#000000"/>
+                    <SolidColorBrush x:Key="ToggleButtonBorderBrushPressed" Color="Transparent"/>
+
+                    <SolidColorBrush x:Key="ToggleButtonBackgroundChecked" Color="#B8B8B8"/>
+                    <SolidColorBrush x:Key="ToggleButtonForegroundChecked" Color="#000000"/>
+                    <SolidColorBrush x:Key="ToggleButtonBorderBrushChecked" Color="Transparent"/>
+
+                    <SolidColorBrush x:Key="ToggleButtonBackgroundCheckedPointerOver" Color="#DADADA"/>
+                    <SolidColorBrush x:Key="ToggleButtonForegroundCheckedPointerOver" Color="#000000"/>
+                    <SolidColorBrush x:Key="ToggleButtonBorderBrushCheckedPointerOver" Color="Transparent"/>
+
+                    <SolidColorBrush x:Key="ToggleButtonBackgroundCheckedPressed" Color="#B8B8B8"/>
+                    <SolidColorBrush x:Key="ToggleButtonForegroundCheckedPressed" Color="#000000"/>
+                    <SolidColorBrush x:Key="ToggleButtonBorderBrushCheckedPressed" Color="Transparent"/>
+
+                    <!-- Button color !-->
+                    <SolidColorBrush x:Key="ButtonBackgroundPointerOver" Color="#DADADA"/>
+                    <SolidColorBrush x:Key="ButtonForegroundPointerOver" Color="#000000"/>
+                    <SolidColorBrush x:Key="ButtonBorderBrushPointerOver" Color="Transparent"/>
+
+                    <SolidColorBrush x:Key="ButtonBackgroundPressed" Color="#B8B8B8"/>
+                    <SolidColorBrush x:Key="ButtonForegroundPressed" Color="#000000"/>
+                    <SolidColorBrush x:Key="ButtonBorderBrushPressed" Color="Transparent"/>
+                </ResourceDictionary>
+                <ResourceDictionary x:Key="HighContrast">
+                    <Style x:Key="FontIconStyle" TargetType="FontIcon">
+                        <Setter Property="FontSize" Value="12" />
+                        <Setter Property="Foreground" Value="{ThemeResource SystemColorWindowTextColor}" />
+                    </Style>
+                    <Style x:Key="SearchBoxBackground" TargetType="StackPanel">
+                        <Setter Property="Background" Value="{ThemeResource SystemColorWindowColor}" />
+                    </Style>
+                </ResourceDictionary>
+            </ResourceDictionary.ThemeDictionaries>
+        </ResourceDictionary>
+    </UserControl.Resources>
+
+    <StackPanel Orientation="Horizontal" Style="{ThemeResource SearchBoxBackground}" Padding="5" CornerRadius="2">

```

Comment:
Let's make the corner radius on the `StackPanel` `0,0,2,2` instead to just round the bottom.